UKHO.D365CallbackDistributorStub.API/UKHO.D365CallbackDistributorStub.API/Services/CallbackService.cs

```diff
@@ -42,5 +54,56 @@ public static bool SaveCallbackRequest(CallbackRequest callbackRequest, string s
             }
 
         }
+
+        public bool SaveCommandCallbackRequest(string subscriptionId, HttpStatusCode? httpStatusCode)
+        {
+            try
+            {
+                CommandCallbackRequest? commandCallbackRequest = s_commandCallbackRequestList.Where(a => a.SubscriptionId == subscriptionId).LastOrDefault();
```

Comment:
Please simplify LINQ expression as below:
```suggestion
                CommandCallbackRequest? commandCallbackRequest = s_commandCallbackRequestList.LastOrDefault(a => a.SubscriptionId == subscriptionId);
```

Reply:
Fixed